docs/docs/administration/backup-and-restore.md

```diff
@@ -64,34 +88,76 @@ gunzip < db_dumps/last/immich-latest.sql.gz | docker exec -i immich_postgres psq
 Immich stores two types of content in the filesystem: (1) original, unmodified content, and (2) generated content. Only the original content needs to be backed-up, which includes the following folders:
 
 1. `UPLOAD_LOCATION/library`
-1. `UPLOAD_LOCATION/upload`
-1. `UPLOAD_LOCATION/profile`
+2. `UPLOAD_LOCATION/upload`
+3. `UPLOAD_LOCATION/profile`
+
+### Asset Types and Storage Locations
+
+<Tabs>
+  <TabItem value="storage template engine OFF (Default)." label="storage template engine OFF (Default)." default>
+
+:::note
+`\library\library` folder is not used by default on new machines running version 1.92.0. These are if the system administrator activated the storage template engine, for [more info](https://github.com/immich-app/immich/releases#:~:text=the%20partner%E2%80%99s%20assets.-,Hardening%20storage%20template,-We%20have%20further).
+:::
 
 **1. User-Specific Folders:**
 
 - Each user has a unique string representing them.
-  - The main user is "Admin" (but only for `\library\library\`)
+- You can find your user ID in Account Account Settings -> Account -> User ID.
+
+**2. Asset Types and Storage Locations:**
+
+- **Source Assets:**
+  - Original assets uploaded through the browser interface & mobile & CLI.
+  - Stored in `/library/upload/<userID>`.
+- **Avatar Images:**
+  - User profile images.
+  - Stored in `/library/profile/<userID>`.
+- **Thumbs Images:**
+  - Preview images (blurred, small, large) for each asset and thumbnails for recognized faces.
+  - Stored in `/library/thumbs/<userID>`.
+- **Encoded Assets:**
+  - By default, unless otherwise specified re-encoded video assets for wider compatibility.
+  - Stored in `/library/encoded-video/<userID>`.
+
+</TabItem>
+  <TabItem value="storage template engine ON" label="storage template engine ON">
+
+:::note
+If you choose to activate the storage template engine, it will move all assets to `/library/library/<userID>`.
+
+When you turn off the storage template engine, it will leave the assets in `/library/library/<userID>` and will not return them to `/library/upload`.  
+**New assets** will be saved to `/library/upload`.
+:::
+
+**1. User-Specific Folders:**
+
+- Each user has a unique string representing them.
+  - The main user is "Admin" (but only for `/library/library/`)
   - Other users have different string identifiers.
-- You can find your user ID in Account Account Settings > Account > User ID.
+- You can find your user ID in Account Account Settings -> Account -> User ID.
 
 **2. Asset Types and Storage Locations:**
 
 - **Source Assets:**
-  - Original assets uploaded through the browser interface&mobile&CLI.
-  - Stored in `\library\library\<userID>`.
+  - Original assets uploaded through the browser interface & mobile & CLI.
+  - Stored in `/library/library/<userID>`.
```

Comment:
I don't think /`library/library` path is incorrect, I think what you mean is `UPLOAD_LOCATION/library` correct?

Reply:
Yes you are right, my habit was to use the default directory as it appears on my end.
Thanks for your attention, fixed!